config/locales/es.yml

```diff
@@ -102,6 +107,8 @@ es:
   passed: Todo en orden
   passed_with_warnings: Funcionó, pero hay cosas para mejorar
   pending: pendiente
+  pending_messages: Tenés mensajes sin leer
+  pending_messages_explanation: No vas a poder enviar nuevas soluciones hasta no leer los mensajes pendientes
```

Comment:
I 'd try to put this in a more informal way. E.g.: "No vas a poder seguir enviando soluciones hasta que no leas los mensajes que te enviaron" or "No vas a poder seguir enviando soluciones hasta que no leas los mensajes de tu docente" o "¡Tenés mensajes sin leer! Revisalos antes de volver de seguir enviando soluciones" 

@faloi what do you think?